dags/kpi_forecasting.py

```diff
@@ -51,6 +51,29 @@
         dag=dag,
     )
 
+    kpi_forecasting_desktop_non_cumulative = gke_command(
+        task_id="kpi_forecasting_desktop_non_cumulative",
+        command=[
+            "python", "kpi-forecasting/kpi_forecasting.py",
+            "-c",
+        ] + ["kpi-forecasting/yaml/desktop_non_cumulative.yaml"],
+        docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/kpi-forecasting_docker_etl:latest",
+        dag=dag,
+    )
+
+    wait_for_desktop_usage = ExternalTaskSensor(
+        task_id="wait_for_desktop_usage",
+        external_dag_id="bqetl_main_summary ",
+        external_task_id="telemetry_derived__firefox_desktop_usage__v1 ",
```

Comment:
whoops, some stray spaces we should remove
```suggestion
        external_dag_id="bqetl_main_summary",
        external_task_id="telemetry_derived__firefox_desktop_usage__v1",
```